.travis.yml

```diff
@@ -20,6 +20,7 @@ matrix:
        - CC=gcc-8
        - CXX=g++-8
        - ARGS="--with-flux-security --enable-caliper"
+       - DISTCHECK_CONFIGURE_FLAGS="--with-flux-security --enable-caliper"
        - DISTCHECK=t
```

Comment:
BTW, by default environment variables are not passed inside the docker container unless they are explicitly passed via `-e VAR` to `docker run` in `docker-run-tests.sh`. Sorry, that is a bit confusing.